Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManager.cs

```diff
@@ -123,9 +118,9 @@ private void PlayEvent(string eventName, GameObject emitter, AudioSource primary
             }
 
             // If the instance limit has been reached...
-            if (currentEvent.instanceLimit != 0 && GetInstances(eventName) >= currentEvent.instanceLimit)
+            if (currentEvent.InstanceLimit != 0 && GetInstances(eventName) >= currentEvent.InstanceLimit)
             {
-                if (currentEvent.instanceBehavior == AudioEventInstanceBehavior.KillNewest)
+                if (currentEvent.AudioEventInstance == AudioEventInstanceBehavior.KillNewest)
```

Comment:
nit: this name change removes the word "Behavior", causing it to be less accurate and less clear what it's referring to.